src/routes/questions.js

```diff
@@ -0,0 +1,23 @@
+const express = require('express');
+const questionRouter = express.Router;
+const questionModel = require('../components/questions/questionModel');
+
+//post questions after register
+questionRouter.post('/register-question', (req, res, next) => {
+    questionModel.registerQuestions.create(req.body).then((question) => {
+        res.send({
```

Comment:
Okay. I'll merge this. But don't forget to change your response to use JsendSerializer as in AuthController